### DISCUSSION TITLE
Values of Form.List

### DISCUSSION THREAD

**Discussion by Archisketch-Finn** [ant-design/ant-design#48164] [Q&A] [answered]
When i use Form.List and submit, i receive values as bellow. How can receive values with only fields as Form.Item's name?

![image](https://github.com/ant-design/ant-design/assets/105255382/1e39de2a-c1f1-467b-81f9-11f0261af605)

**Comment by nouman2075** [comment, marked as answer]
Where you call Form.List `add` function, you pass `add` to `onClick` directly like this `onClick={add}`. You need to call `add` function like this `onClick={() => add()}`

**Reply by Archisketch-Finn** [reply]
Thank you @nouman2075
